README-clearwater.md

```diff
@@ -0,0 +1,5 @@
+## Clearwater build instructions
+
+To build this package from a fresh checkout, run `debuild --no-lintian -us -uc -b`.
```

Comment:
Where does this build the package to? The current directory? Is there any way to change that? That's fine, I was just wondering and if there's anything of interest here then it would be worth documenting.